actions/assembler/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Documentation Assembler'
+description: 'Run the documenation assembler commands'
+
+branding:
+  icon: 'filter'
+  color: 'red'
```

Comment:
nit: This is not really required. And I think you can only ever see this if it was published in the marketplace